create_secrets.sh

```diff
@@ -5,6 +5,7 @@ source ./envfile
 # Create namespaces where the secret will be
 oc new-project hive
 oc new-project artifactory
+oc new-project nexus
```

Comment:
I think we can enable automatic namespace creation in Argocd. No need to create the nexus namespace in this script.

Reply:
removed